### ISSUE TITLE
Add a release

### ISSUE THREAD

**Issue by ericsagnes** [NixOS/hydra#373]
It is a pity that Hydra has no official release yet.

As NixOS and many other demanding projects use it daily, it should be stable enough to have a release, or at least a beta release.

I suppose that there are reasons for the lack of release, so I propose the creation of a list of blocker issues (if any) and a roadmap (if required) for a release.

It would be particularly nice if there could be an official release available when NixOS 16.09 get released as it will be the first stable NixOS with an Hydra module.

**Comment by domenkozar** [comment]
See also https://github.com/NixOS/hydra/pull/272

**Comment by phile314-isska** [comment]
Any chance of getting a release sometime soon? I could help with beta testing.

**Comment by domenkozar** [comment]
FYI: there is now hydra package and module in nixpkgs.

**Comment by phile314-isska** [comment]
Let me rephrase my comment, it would be nice to have a newer hydra version in nixpkgs ;-) . I am using the Hydra version in Nixpkgs at the moment. I want to use a newer version, but it's kind of difficult to figure out which Hydra commits are (somewhat) stable from the git history alone.

**Comment by domenkozar** [comment]
Try https://github.com/NixOS/nixpkgs/pull/19396
